draft-meynell-panrg-scion-research-questions.md

```diff
@@ -196,9 +196,34 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 
 ## ISD, AS identity
 
+The SCION protocol specifies 16 bits and 48 bits to identify the ISD and AS.
+These identification is, at this moment, necessary at the data-plane level,
+written on each one of the data packets to fully address the sender and receiver,
+as well as at the control-plane level, to identify the PCB sender and hops.
+
+While 48 bits seem sufficient to accommodate a large number of ASes,
+using 16 bits for the ISD might not be.
+Additionally, the justification of these figures is lacking at the moment.
+
+The following questions arise: (not comprehensive)
+
 * How many ASes do we expect in the SCION network model?
-* How many ISDs? What is the ontology of an ISD? Per geographic area only?
-* One AS belongs to many ISDs?
+* Can one AS belong to many ISDs?
+* Are AS numbers unique themselves? Or only unique in combination with an ISD?
+* How many ISDs do we expect?
+* What is the ontology of an ISD?
+  Note that not all combinations of ASes might be possible as ISDs:
+  An ISD MUST be connected via at least one core link to the rest of the ISD connected graph.
+  Otherwise the beaconing process does not transfer trust information among
+  the connected graph of existing ISDs and this ISD in question.
+  This reduces the absolute number of possible ISDs to those that satisfy a
+  strict direct connectivity requirement (hence they could become core ASes in
+  the new ISD). It is still super-exponential asymptotically.
+  * Per geographic area?
+  * Per legal jurisdiction?
+  * Per capacity tier?
```

Comment:
Can you briefly motivate what the benefit would be for someone to create an ISD to indicate capacity?
Also, since an AS has a given capacity, I guess it should only ever be in one of those ISDs, never in multiple ones?

Reply:
> but there might be many e.g. "Tier-2" ISDs, since not all ASes that fall into the category tier-2 can directly connect with each other.

Is that the case? I didn't think a direct connection is a requirement? For example in SCIERA, I guess not all Universities or core ASes have a direct connection, even if they are all in the same ISD?

That question aside, I am still not sure what the motivation would be to create ISDs for capacity tiers. 
However, I won't object if you leave it as is :-)